### PR TITLE
fixes issues #5374 and #5504

### LINKED ISSUE #5374: radio.selected(value) does not work with radio buttons holding "value" and "label"
#### Most appropriate sub-area of p5.js?

- [ ] Accessibility (Web Accessibility)
- [ ] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [x] DOM
- [ ] Events
- [ ] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Details about the bug:

- p5.js version: 1.4.0
- Web browser and version:  Chrome and 92.0.4515.131 (Official Build)  (64 bit) (cohort: Stable)
- Operating System: Windows 10
- Steps to reproduce this:

Hi all,
In the following code, the "red" button is not selected. Am I missing something?
```
let radio; 

function setup() {
  radio = createRadio(); 
  radio.option(1, "red"); 
  radio.option(2, "blue"); 
  radio.option(3, "green");
  radio.selected(1); 
}

function draw() {

}
```

### PR DESCRIPTION
**Pull request by reejuBhattacharya**
Resolves #5374 and #5504 

 Changes:
Updated the documentation of `createRadio()`

 Screenshots of the change:
Before: 
image
image

After:
image
image

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by reejuBhattacharya** [comment]
@limzykenneth @outofambit 
Would love it if you could check out my pull request. Great thanks!

File: src/dom/dom.js [hunk 1]
```diff
@@ -772,11 +772,14 @@ p5.prototype.createSelect = function() {
...
  *   element. If an option with a value already exists, it will be returned.
+ *   It is recommended to use string values as input for `value`.
  *   Optionally, a label can be provided as second argument for the option.
- * - `.remove(value)` can be used to remove an option for the element.
+ * - `.remove(value)` can be used to remove an option for the element. String
+ *   literals recommended as input for `value`.
  * - `.value()` method will return the currently selected value.
  * - `.selected()` method will return the currently selected input element.
- * - `.selected(value)` method will select the option and return it.
+ * - `.selected(value)` method will select the option and return it. String
```
Comment (limzykenneth) [inline src/dom/dom.js:781]: For here and for `.remove(value)` above, string literals is not the recommendation but rather just strings as string literals have a specific meaning of only including strings defined inline with quotemarks and not string variables.

**Review by limzykenneth — CHANGES_REQUESTED** [review]

File: src/dom/dom.js (hunk 1 above)
Comment (reejuBhattacharya) [reply src/dom/dom.js:781]: Thanks for the check! I have changed "string literals" to "string values".

**Review by limzykenneth — APPROVED** [review]

**Comment by limzykenneth** [comment]
Looks good. Thanks!

**Pull request merged**